lms/djangoapps/discussion/django_comment_client/utils.py

```diff
@@ -465,7 +465,7 @@ def get_discussion_category_map(course, user, divided_only_if_explicit=False, ex
             )
         }
 
-    _sort_map_entries(category_map, course.discussion_sort_alpha)
+    _sort_map_entries(category_map, False)
```

Comment:
Is this the only place this method is used? and why not remove the param instead of hardcoding false?

Reply:
I think removing param completely will make test cases fail on many other places (not just for utils.py).